### ISSUE TITLE
Script starts and gets stuck on servant details

### ISSUE THREAD

**Issue by ceryskies** [Fate-Grand-Automata/FGA#1041]
### Preparation

- [X] I tested the [latest release](https://github.com/Fate-Grand-Automata/FGA/releases/tag/canary)
- [X] I looked at other issues (even the [closed ones](https://github.com/Fate-Grand-Automata/FGA/issues?q=is%3Aissue+is%3Aclosed))
- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)

### FGO server

NA

### FGA build number

1592

### Describe the bug

hello hello,,, i um dont really understand all this code stuff and I feel really dumb but man I don't know why is it doing this...

basically whenever I start the script, it opens the servant detail page to do the matching, and then it gets stuck there. i never had this issue before as before, it would open the details, then close it shortly after. I even tried updating to the newest release and that didn't help either. 

sorry >_< 

### Video

https://youtu.be/mDg7kxml_jw & https://youtu.be/2nvEVVjCmJU
![cattoshook](https://user-images.githubusercontent.com/6912408/147294333-146a0eed-b5ae-4e16-a22c-3a796d4cbcec.png)

### Device model

Pixel 4a

### Android version

12

**Comment by Kyonyuu** [comment]
oh, so it's happening to everyone... it's happening to me tho, but in cn... it's not always but it has a chance to(i'm using nox, and my pc is new...)

**Comment by Kyonyuu** [comment]
#1038 my issue, i had just recorded a video, this bug for some reason likes attention so it most of the time don't work while recording.

**Comment by Kyonyuu** [comment]
just watched the video, and the bug is different... btw recomend go to the options and turn off the option to do brave chain, won't fix but will at least workaround... maybe...

**Comment by reconman** [comment]
Duplicate of #1039

**Comment by reconman** [comment]
Sorry, not a duplicate.

**Comment by reconman** [comment]
So I started Bluestacks with your exact resolution and it closes the detail window normally by clicking in the top right. In your video, I don't see any click there.

Can you try increasing the "wait after click" delay in More Options -> Advanced -> Fine-tune -> Click?

It's already waiting 250 milliseconds before taking the screenshot, but I have no idea what else it could be.

**Comment by ceryskies** [comment]
the delay was already at 450 MS D:

**Comment by reconman** [comment]
I've got one more idea. Did you record the videos with the camera hole in the bottom left or top right?

If it was top right, maybe the camera hole is blocking inputs? Then rotating the screen by 180 degrees would help.

It doesn't cause issues on my device, but you never know.

**Comment by ceryskies** [comment]
ah that fixed it!! thank you so much T_T
